python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_restraint.py

```diff
@@ -311,86 +346,244 @@ def write_dihedral(key_list, equilibrium_values, force_constants):
 
         return "\n".join(output)
 
-    def toString(self, engine="Gromacs"):
-        """
-        The method for convert the restraint to a format that could be used
+    def _somd_boresch(self):
+        '''Format the SOMD string for the Boresch restraints.'''
```

Comment:
Can this be switched to double quotes, i.e. `"""`. I imagine this would be changed by `black`.